Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +35,19 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            var temp = 0;
+            for (var i = 0; i < array.Length; i++)
+            {
+                if (i == array.Length / 2) break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4899409)